tests/test_storage_macro.rs

```diff
@@ -51,6 +84,16 @@ async fn test_storage_macro_chunk_redis(
     Ok(())
 }
 
+async fn get_colink_user() -> Result<
```

Comment:
naaaaaaaming - this is definitely more than getting a user
(1) can we just make a fixture out of this? As it is used in all tests?
(2) `set_up_test_env`?